h2o-r/h2o-package/R/automl.R

```diff
@@ -317,6 +317,7 @@ h2o.automl <- function(x, y, training_frame,
   # GET AutoML object
   aml <- h2o.get_automl(project_name = res$job$dest$name)
   attr(aml, "id") <- res$job$dest$name
+  attr(aml, '_build_json') <- res # hidden attribute for debugging/testing
```

Comment:
@sebhrusen Just trying to understand couple of things: 
1) so we did not have _build_resp analog in R? In Py do we use `_build_resp` for non testing purposes? 
2) why it is called `_build_json` here. _build_resp is derived from the same json but parsed into object and in case of R we don't have it parsed, right?  
3) Are we extending API just for testing purposes?

Reply:
in Python, variables prefixed with `_` are not considered as API: it's a naming convention to say they are "private" and using them in consumer code is not safe.
in R, it's a bit similar with attributes, they're not considered as API (they don't appear in class description), but rather some kind of metadata.
I'll rename to `_build_resp` for consistency with Py

Reply:
@sebhrusen good then, thanks for the info!